save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestSuitesController.kt

```diff
@@ -16,10 +18,10 @@ class TestSuitesController {
     private lateinit var testSuitesService: TestSuitesService
 
     /**
-     * @param testSuite
+     * @param testSuiteDtos
+     * @return mono list of TestSuite
      */
-    @PostMapping("/saveTestSuite")
-    fun saveTestSuite(@RequestBody testSuite: TestSuite) {
-        testSuitesService.saveTestSuite(testSuite)
-    }
+    @PostMapping("/saveTestSuites")
+    fun saveTestSuite(@RequestBody testSuiteDtos: List<TestSuiteDto>): Mono<List<TestSuite>> =
+            Mono.just(testSuitesService.saveTestSuite(testSuiteDtos))
```

Comment:
This line is marked as not covered with tests

Reply:
There are two tests (`testConnection`, `checkDataSave`) for this request.